tests/dataframe_tests/test_dataframe_aggregation.py

```diff
@@ -87,8 +90,33 @@ def test_translate_count_aggregation(translator):
     )
     result_df = translator.aggregate_nested_properties()
 
-    print(result_df.show(truncate=False))
     assert_dataframe_equality(expected_df, result_df, get_node_id_name("imaging_study"))
     diff = []
     assert_zero(result_df, diff, ["_dx_series_file_count", "_mr_series_file_count"])
     assert diff == [], f"Differences: {diff}"
+
+@pytest.mark.schema_parent
+@pytest.mark.parametrize("translator", [("parent", "participant", "aggregation", [])], indirect=True)
+def test_flatten_nested_array_parent_props(translator):
+    """
+    Test to ensure the created dataframe will not contains any array being nested in another array
+    - input dataframe with leaf nodes containing nested list
+    - expected dataframe with leaf nodes no longer containing nested list
```

Comment:
it's hard to understand this without examples, could you add some plaintext data in the docstring? it can just be a sample of what's in the input and output dataframes

Reply:
I will add some docstring

Reply:
could you add actual example fields+values